src/Native/System.Native/pal_networkchange.cpp

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#include "pal_config.h"
+
+#if HAVE_LINUX_RTNETLINK_H
```

Comment:
I wasn't able to get this to work for whatever reason. The file didn't seem to actually be included in compilation although I could tell the sources were being set (I printed them out after this block)